qtpy/QtWidgets.py

```diff
@@ -22,6 +22,110 @@
 from qtpy import PythonQtError
 
 
+def patch_qcombobox():
+    """
```

Comment:
I'd prefer this function to leave in a different file. It could be called `pyside_combobox` or something like that :-)

@Nodd, @goanpeca, what do you think?